integrations/helpers_test.go

```diff
@@ -20,6 +20,14 @@ import (
 	"github.com/influxdata/kapacitor/uuid"
 )
 
+func mustParseTime(s string) time.Time {
```

Comment:
Looks like this is defined twice?

Reply:
It exists in integrations test and in a different tests.  I went the coping route, rather than creating a helper package for one tiny function that only exists to make tests easier to read.